inventory/acc

```diff
@@ -12,6 +12,10 @@ acc62.coin.surf.net
 acc41.coin.surf.net
 acc42.coin.surf.net
 
+[php56-apps]
+acc41.coin.surf.net
+acc42.coin.surf.net
```

Comment:
This cannot occur as with the current setup running two different php versions on the same machine is not supported. Which machine(s) this should be should be requested from SURFnet (ping @thijskh @quartje) or, if now unkown, should be removed completely until known.